doc/content/integrations/adding-oauth-clients/_index.md

```diff
@@ -0,0 +1,194 @@
+---
+title: "Adding OAuth Clients"
+description: ""
+weight: -1
+aliases:
+  [ /integrations/adding-oauth-clients ]
+---
+
+This section contains instructions for creating an OAuth Client.
+
+<!--more-->
+
+{{% tts %}} uses the [OAuth 2.0 protocol](https://oauth.net/) for authentication and authorization.
+
+#### Considerations
+
+When you register an OAuth client, you should keep in mind that it will have to be approved by an admin. The scope and duration of the review process depends on the network where you register your OAuth client. If you register it in your own tenant on The Things Stack Cloud, then the administrator (probably you) can approve it right away. If you register an OAuth client on {{% ttss %}}, the administrators will take some time to review if and how the OAuth client benefits the community. 
```

Comment:
```suggestion
When you register an OAuth client, you should keep in mind that it will have to be approved by an administrator. The scope and duration of the review process depends on the network where you register your OAuth client. If you register the OAuth client in {{% tts %}} Cloud, Enterprise or Open Source, then the administrator (probably you) can approve it right away. If you register an OAuth client on {{% ttss %}}, the administrators will take some time to review if and how the OAuth client benefits the community. 
```